cool-core/src/main/java/com/nus/cool/functionality/CohortExploration.java

```diff
@@ -0,0 +1,36 @@
+package com.nus.cool.functionality;
+
+import java.io.IOException;
+
+import com.nus.cool.core.io.readstore.CubeRS;
+import com.nus.cool.core.io.storevector.InputVector;
+import com.nus.cool.core.util.writer.CliDataWriter;
+import com.nus.cool.core.util.writer.DataWriter;
+import com.nus.cool.model.CoolModel;
+
+public class CohortExploration {
+  public static void main(String[] args) {
+    String datasetPath = args[0];
+    String cubeName = args[1];
+    String cohortName = args[2];
+    try {
+      CoolModel coolModel = new CoolModel(datasetPath);
+
+      // load cube
+      coolModel.reload(cubeName);
+      CubeRS inputCube = coolModel.getCube(cubeName);
+
+      // load cohort
+      coolModel.loadCohorts(cohortName, cubeName);
+      InputVector userVector = coolModel.getCohortUsers(cohortName);
+
+      // export cohort
+      DataWriter writer = new CliDataWriter();
```

Comment:
We should add an object for the writer to add data instead of just displaying the data.

Reply:
Beside, add one more writer to write these data into a new cube,

Reply:
Added a ListDataWriter that put the records as string into a list. The old CliDataWriter (Command line interface Datawriter) stay unchanged to demonstrate the use in cool-functionalities